core/sql/regress/hive/EXPECTED007

```diff
@@ -353,9 +355,7 @@ A            B
 
 >>unregister hive table if exists hive.hive.thive1;
 
-*** ERROR[4263] Object HIVE.HIVE.THIVE1 has invalid state and cannot be accessed. It is registered in trafodion metadata but the correponding hive object does not exist. Use 'cleanup hive table/view' command to cleanup this object.
-
-*** ERROR[3251] UNREGISTER operation could not be completed.
+*** ERROR[3251] UNREGISTER operation could not be completed. Reason: Specified object THIVE1 does not exist.
```

Comment:
Good eye.
The error message is incorrect. It should return error that table has invalid state and need to be cleaned up.
Check for 'if exists' only apply to tables with valid state.
Will fix this in a subsequent release.